### ISSUE TITLE
Unrecognized command: git upload-pack when cloning existing repository

### ISSUE THREAD

**Issue by bindu-rk** [git-for-windows/git#1263]
- [ ] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?
64-bit
```
$ git --version --build-options
2.14.1
** insert your machine's response here **
```
I am getting the following error when cloning an existing repository. Is it missing a hyphen
Unrecognized command: git upload-pack '/....'
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?
Windows 10
```
$ cmd.exe /c ver

** insert your machine's response here **
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?
I chose defaults

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

** insert your machine's response here **
```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

** insert your response here **

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other
Bash
** insert your response here **

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.
```
** insert your commands here **
git clone url
```
 - What did you expect to occur after running these commands?

** insert here **
expected the repository to be cloned
 - What actually happened instead?

** insert here **
got the error Unrecognized command: git upload-pack '/
 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?
The repository is available for over a year, I had to clone since I got a new laptop

** insert URL here **

**Comment by dscho** [comment]
> ** insert here **

BTW these were intended to be *replaced* by your information.

> $ git --version --build-options
> 2.14.1
> ** insert your machine's response here **

That can't be right. `--build-options` prints a lot more information. And Git for Windows certainly never reports anything like `v2.14.1`. If at all, it reports `v2.14.1.windows.1`.

> Unrecognized command: git upload-pack '/....'
> fatal: Could not read from remote repository.

This sounds awfully like the bug I introduced into v2.14.*0* and fixed in v2.14.0(2) already.

Are you really, 100% certain that you tested the correct version?

**Comment by bindu-rk** [comment]
Thank you for your quick response. I am sorry I did not realize the version had changed.
I had 2.14.0, I downloaded the latest version and was able to successfully clone the repository.
